buildroot/arm64.config

```diff
@@ -1,7 +1,9 @@
 BR2_aarch64=y
 BR2_cortex_a57=y
 BR2_OPTIMIZE_3=y
-BR2_TOOLCHAIN_EXTERNAL=y
```

Comment:
any idea why this was set for arm64?

Reply:
Was just the config I used when I first got into buildroot, I didn't want to build a toolchain in addition to the rootfs image.